### ISSUE TITLE
Complex coefficients

### ISSUE THREAD

**Issue by billmclean** [jverzani/SpecialPolynomials.jl#26]
I am trying to create a sum of Hermite polynomials with complex coefficients, but the obvious approach does not work.

```
julia> x=variable(Polynomial{ComplexF64})
Polynomial(1.0*x)

julia> p=Hermite([1.0+2.0im,-0.5,3.7im, 0.1+0.01im])
Hermite(2.23606797749979⋅H₀(x) -0.5⋅H₁(x) + 3.7⋅H₂(x) + 0.1004987562112089⋅H₃(x))
```
Is the only option to define the real and imaginary parts of the sum separately?

**Comment by jverzani** [comment]
That is definitely a bug. I have some time tomorrow for looking into it. Thanks for alerting me to the issue!

**Comment by jverzani** [comment]
In case you are playing along at home in real time, it appears to be a bug in `show`; the underlying polynomials has complex coefficients and hopefully behaves as you expect.

**Comment by jverzani** [comment]
closed by #27
